content/documentation/glossary/index.adoc

```diff
@@ -0,0 +1,207 @@
+
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
+link:https://istio.io/help/glossary/#deployment[Istio Glossary]
+
+== Destination
+
+The remote upstream service Envoy is talking to on behalf of a source workload. There can be one or more service versions for a given service and Envoy chooses the version based on routing rules.
+
+link:https://istio.io/help/glossary/#destination[Istio Glossary]
+
+=== Destination Rule
+
+DestinationRule defines policies that apply to traffic intended for a service after routing has occurred. These rules specify configuration for load balancing, connection pool size from the sidecar, and outlier detection settings to detect and evict unhealthy hosts from the load balancing pool
+link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#DestinationRule[Istio Destination Rule]
+
+
+== Endpoints
+
+A communication endpoint is a type of communication network node. It is an interface exposed by a communicating party or by a communication channel
+
+== Envoy
+
+The high-performance proxy that Istio uses to mediate inbound and outbound traffic for all services in the service mesh. Learn more about Envoy.
+link:https://istio.io/help/glossary/#envoy[Istio Glossary]
+
+== Envoy Health
+
+It's the status of the incoming and outcoming traffic.
+
+== Error Rate
+
+it's the percentage the errors in the traffic to a specific object in a <<Rate Interval>>
+
+== Gateway
+
+Gateway describes a load balancer operating at the edge of the mesh receiving incoming or outgoing HTTP/TCP connections. The specification describes a set of ports that should be exposed, the type of protocol to use, SNI configuration for the load balancer, etc.
+link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Gateway[Istio Gateway]
+
+== Inbound Metrics
+
+Incoming Metrics to a <<Workload>>, <<Service>> or <<Application>>
+
+== Istio Config
+
+It's a configuration object in a Istio environment
+
+=== Istio Name
+
+It's the name of the <<istio Config>>
+
+=== Istio Type
+
+It's the type of the <<Istio Config>>, could be <<Gateway>>, <<VirtualService>>, <<DestinationRule>>, <<ServiceEntry>>, <<Rule>>, <<QuotaSpec>> or <<QuotaSpecBinding>>.
+
+== Istio Sidecar
+
+This represents if the Istio Object has link:https://istio.io/docs/reference/commands/sidecar-injector/[Istio Sidecar] deployed.
+
+== Quota
+
+A limited or fixed number or amount of resources.
+
+== QuotaSpec
+
+File that define a <<Quota>>
+
+== QuotaSpecBinding
+
+Quota spec binding is used to bind link:https://istio.io/help/faq/mixer/[mixer] quota resource name to a proxy.
+
+== Label
+
+It's a tag. The client/user can identify a set of objects with this.
+
+An empty label selector (that is, one with zero requirements) selects every object in the collection.
```

Comment:
a Label selector is another concept, and is set separately on objects from labels. it's worth to explain it separately/link to explanation in other docs.